client/types.go

```diff
@@ -59,6 +59,57 @@ const (
 	ConnectConsistencyLevelAll
 )
 
+// String returns the consistency level as a string
+func (l ConnectConsistencyLevel) String() string {
+	switch l {
+	case ConnectConsistencyLevelAny:
+		return "ConnectConsistencyLevelAny"
+	case ConnectConsistencyLevelNone:
+		return "ConnectConsistencyLevelNone"
+	case ConnectConsistencyLevelOne:
+		return "ConnectConsistencyLevelOne"
+	case ConnectConsistencyLevelMajority:
+		return "ConnectConsistencyLevelMajority"
+	case ConnectConsistencyLevelAll:
+		return "ConnectConsistencyLevelAll"
+	}
+	return ""
```

Comment:
Might be better to return "ConnectConsistencyLevelUnknown" so it's easier to debug?